File: challenges/1-produto-simples/python/virusxdrj/produto_simples.py
from sys import argv


def prod(num_1: int, num_2: int) -> None:
    print(num_1 * num_2)


if __name__ == "__main__":
    args: list[str] = argv[1:3]
    args: list[int] = [int(arg) for arg in args]
    prod(args[0], args[1])

File: challenges/1-produto-simples/php/LpxsBr/produto-simples.php
<?php

function produto(int $num1, int $num2){
    
    $PROD = ($num1 * $num2);
    return 'produto: '.($PROD);
}

echo $resp = (isset($argv[1]) and isset($argv[2])) ? produto($argv[1], $argv[2]) : "valores não informados, tente novamente";
